packages/web-react/.eslintrc

```diff
@@ -0,0 +1,60 @@
+{
+  "root": true,
+  "env": {
+    "es6": true,
+    "browser": true
+  },
+  "parser": "@typescript-eslint/parser", // the TypeScript parser we installed earlier
+  "parserOptions": {
+    "project": "./config/tsconfig.json",
+    "ecmaFeatures": { "jsx": true } // Allows for the parsing of JSX
+  },
+  "settings": {
+    "import/resolver": {
+      "node": {
+        "extensions": [".js", ".jsx", ".ts", ".tsx"]
+      }
+    }
+  },
+  "extends": [
+    "eslint:recommended", // eslint default rules
+    "airbnb",
+    "prettier",
+    "plugin:@typescript-eslint/eslint-recommended", // eslint TypeScript rules (github.com/typescript-eslint/typescript-eslint)
+    "plugin:@typescript-eslint/recommended",
+    "plugin:react/recommended", // eslint react rules (github.com/yannickcr/eslint-plugin-react)
+    "plugin:jsx-a11y/recommended", // accessibility plugin
+    // Prettier plugin and recommended rules
+    "plugin:prettier/recommended"
+  ],
+  "plugins": [
+    "standard",
+    "promise",
+    "react"
+  ],
+  "rules": {
+    // Include .prettierrc.js rules
+    "prettier/prettier": [
+      "error", {}, { "usePrettierrc": true }
```

Comment:
all this prettier setup is easier to do when importing eslint-config-prettier package and there is no need to define it differently ..

Reply:
I've refactored this.